packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -416,6 +418,11 @@ class ToggleButtons extends StatelessWidget {
   /// the buttons starting from the first or last child from top to bottom.
   final VerticalDirection verticalDirection;
 
+  /// Whether to expand the children to take height of the tallest widget.
+  ///
+  /// Defaults to false.
+  final bool expandChildren;
```

Comment:
I think I found a way for this to not be necessary. I played with this a bit today and found that all existing and new tests pass if we base this on whether or not constraints are provided, instead of using this flag.

Reply:
```suggestion
```
Without this, the same can be achieved with the following changes

Reply:
This is brilliant, I didn't think about nulling max constraints.

Reply:
Thanks! I can't take too much credit. I migrated all of the customers inside of Google off of FlatButton, RaisedButton and OutlineButton. It took a while to figure out! :)